pipelines/upgrade/05-server_to_intermediate.yaml

```diff
@@ -21,3 +21,11 @@
     - foreman_server_repositories
     - role: foreman_installer
       foreman_installer_skip_installer: "{{ true if forklift_upgrade_version_start == forklift_upgrade_version_intermediate else false }}"
+  post_tasks:
+    - name: Run remove Pulp 2
+      command: foreman-maintain content remove-pulp2
+      when:
+        - forklift_upgrade_version_intermediate == "4.0"
+        - pipeline_type == "katello" or pipeline_type == "luna"
```

Comment:
Generally not a huge fan of hardcoding pipeline type names. Would it be better to set a variable in a vars file about this?